src/components/PageTwo.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
I would suggest creating a separate folder for the pages (PageOne, PageTwo, WelcomePage).  Imagine - you will create some other components such as buttons, inputs, tables, etc, + other new pages and put them into the components folder, very soon everything grows and mixes up, and it will be more difficult to search and maintain. And the logical meaning of components and pages is slightly different, so they should be in different folders. Name this new folder - pages.